### PR TITLE
use new cluster for jormun db

### PR DESCRIPTION
**Pull request by Hoshiyo**
cf : https://github.com/CanalTP/navitia_deployment_conf/pull/253

### CODE REVIEW

**Comment by Hoshiyo** [comment]
wait monday

File: templates/tyr/settings.py.jinja [hunk 1]
```diff
@@ -15,8 +15,11 @@ CELERY_BROKER_URL = 'amqp://{{tyr_broker_username}}:{{tyr_broker_password}}@{{ra
 #URI for postgresql
 # postgresql://<user>:<password>@<host>:<port>/<dbname>
 #http://docs.sqlalchemy.org/en/rel_0_9/dialects/postgresql.html#psycopg2
+{% if env.replica_postgresql_database_host %}
+SQLALCHEMY_DATABASE_URI = 'postgresql://{{tyr_postgresql_user}}:{{tyr_postgresql_password}}@{{replica_postgresql_database_host}}/{{tyr_postgresql_database}}'
```
Comment (antoine-de) [inline templates/tyr/settings.py.jinja]: I don't understand why you use a different variable for the replica

Comment (Hoshiyo) [inline templates/tyr/settings.py.jinja]: cities uses postgresql_database_host

File: fabfile/component/tyr.py [hunk 2]
```diff
@@ -68,6 +68,7 @@ def update_tyr_config_file():
                         'tyr_postgresql_user': env.tyr_postgresql_user,
                         'tyr_postgresql_password': env.tyr_postgresql_password,
                         'postgresql_database_host': env.postgresql_database_host,
```
Comment (antoine-de) [inline fabfile/component/tyr.py]: hum I think it's worth also renaming `postgresql_database_host` to `ed_postgresql_database_host` no ?

If you want to make it retro compatible you might add a property function to `env`

**Pull request merged**